donate/payments/stripe_webhooks.py

```diff
@@ -1,25 +1,27 @@
-from django import forms
+import json
 from django.http import HttpResponse, HttpResponseBadRequest
 from django.utils.decorators import method_decorator
 from django.views.decorators.csrf import csrf_exempt
-from django.views.generic import FormView
+from django.views.generic import View
 
 from .tasks import queue, process_stripe_webhook
 
 
-class StripeWebhookForm(forms.Form):
-    data = forms.TextInput()
-
-
 @method_decorator(csrf_exempt, name='dispatch')
-class StripeWebhookView(FormView):
```

Comment:
Turns out the only reason `FormView` was used for the Braintree webhook endpoint is because Braintree form encodes their webhook payloads. Stripe uses JSON, so the changes here support that.